config/locales/en.yml

```diff
@@ -685,6 +693,34 @@ en:
       record: Record vaccinations
       register: Register
       triage: Triage
+    dates:
+      label: Session dates
+    reminders:
```

Comment:
Generally we try and make this key match the view structure, which then allows you to use the shorthand syntax.

For example, in the view `sessions/manage_consent_reminders/show.html.erb` you can use `t(".title")` as long as the key `sessions.manage_consent_reminders.show.title` exists.

You should be able to find examples of where we do this across the service.